lib/srv/desktop/windows_server.go

```diff
@@ -164,6 +164,9 @@ type WindowsServiceConfig struct {
 	// Windows Desktops. If multiple filters are specified, they are ANDed
 	// together into a single search.
 	DiscoveryLDAPFilters []string
+	// DiscoveryLDAPAttributeLabels are optional LDAP attributes to convert
+	// into Teleport labels.
+	DiscoveryLDAPAttributeLabels []string
```

Comment:
Maybe `DiscoveryLDAPLabelAttributes` to match the field name in the discovery configuration?